doc/ref_cert/README.md

```diff
@@ -1,5 +1,41 @@
+
+The overall workstream requires the close coordination of the following:
+
+**Requirements** - The agreed upon capabilities and conditions that a compliant NF or cloud infrastructure must provide or satisfy. All requirements will be hosted and maintained in the RA.
+**Tests** - The verification mechanism that determines that a given NF or cloud infrastructure complies with one or more requirements.
+**Conformance Specifications** - The definition of the requirements, tests, and circumstances (test case integration, etc.) that must be met to be deemed conformant.
+
+If there is no clear traceability and strong links between these 3 components, then it becomes difficult to determine if a NF or cloud infrastructure is compliant. With this in mind, below are the set of recommended principles for each of the three components to follow. Adherence to these principles will provide the following:
+
+* Enable clear progress tracking and linkage between independent projects (i.e. know what has and hasn't been covered, and track changes over time)
+* Help users better understand mandatory requirements
+* Provide a stable set of point-in-time requirements and tests to achieve conformance.  
+* Reduce ambiguity in testing, requirements, and conformance
+
+
+## Requirement Principles
+Requirements Principles can be found in the [RM Principles](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#2.0)
+
+## Testing Principles
+* There must be traceability between test cases and requirement being validated
+* Failures should provide additional content to inform the user where or how the requirement was violated (ex: which file or resource violated the requirement).  Put another way, don’t require the user to read the test to understand what went wrong.
+* Testing tools should support selection of tests based on category or profile.
+* Result reports must clearly delineate violations of mandatory vs. optional requirements
+* Tests must be available to run locally by both CNF and NFVI providers
+* Testing tools must produce machine-readable result formats that can be used as input into the badging program (OVP already defines a format).
+
+## Conformance Specifications
+* Conformance specifications must refer to or define the *versioned* requirements that must be satisfied
+* Conformance specifications must refer to the *versioned* test implementations that must be used to validate the requirements
+* Conformance specifications must define the expected preconditions and environment requirements for any test tooling
+* Conformance specifications must define which tests must be executed in the given testing tools to achieve conformance
+* The conformance specification must provide the mapping between tests and requirements to demonstrate traceability and coverage.
+
 # CNTT Reference Conformance (RC) Test Case Integration
```

Comment:
The paragraph that follows needs to be edited

Reply:
which part of it? Would it be more related to #1725 ?

Reply:
@xmulligan Lines 41 to 49. 
" ... must fulfill the following test case integration requirements ..." -- what does "following refer to?
"... guarantee a smooth overall integration" -- can we delete "smooth" as not sure what it "would look like"
"The new RC2 stream asks to move them in this common place applicable to all CNTT conformance suites." -- to move "what"? "in this common place" - what is "this common place" referring to.

Reply:
I think that is outside the scope of this PR since the paragraph you are referring to is from #1725. Can you open a conversation there or issue?

Reply:
@collivier can you help here

Reply:
@pgoyal01 I created #1863 to cover this so I will resolve the conversation now